packages/ui/package.json

```diff
@@ -4,12 +4,12 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "assets:native": "mkdir -p dist-native/src && cp -r src/assets-native ./dist-native/src/",
-    "assets:web": "mkdir -p dist-web/src && cp -r src/assets-web ./dist-web/src",
+    "assets:native": "mkdir -p dist-native/src && cp -r src/assets-native ./dist-native/src && cp -r src/assets ./dist-native/src",
+    "assets:web": "mkdir -p dist-web/src && cp -r src/assets-web ./dist-web/src && cp -r src/assets ./dist-web/src",
```

Comment:
Not a fan of having these copy commands in the package.json. tsup doesn't have a "beforeBuild" like it does `onSuccess` but [per comment here](https://github.com/egoist/tsup/issues/1094#issuecomment-1987318528), anything in the build script runs before the config.

So, could we move these directory/asset copying actions there, using a node script to do the same?

Reply:
I meant to come back to this bc came across a suggestion to use this plugin to copy assets in the tsup config:
https://github.com/LinbuduLab/esbuild-plugins/tree/main/packages/esbuild-plugin-copy